modules/load-balancers/ddos.tf

```diff
@@ -0,0 +1,136 @@
+locals {
+  global_accelerator_hosted_zone_id = "Z2BJ6XQ5FK7U4H" # Fixed Hosted Zone ID for Global Accelerator
```

Comment:
should use the fips endpoint when fips is enabled.
https://docs.aws.amazon.com/general/latest/gr/global_accelerator.html#global_accelerator_region